<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordTabView</class>
 <widget class="QWidget" name="PasswordTabView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="pwdView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="buttonAddPwd">
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <item>
            <property name="text">
             <string>Pas de tri</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tri par utilisation</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Tri alphabétique</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="pwdBox">
       <property name="title">
        <string>Nom du mot de passe</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Mot de passe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblPwd">
            <property name="text">
             <string>***********</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="buttonCopyPwd">
            <property name="text">
             <string>Copier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonUpdatePwd">
            <property name="text">
             <string>Modifier</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Nom d'utilisateur</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblUser">
            <property name="text">
             <string>***********</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="buttonCopyUser">
            <property name="text">
             <string>Copier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="buttonUpdateUser">
            <property name="text">
             <string>Modifier</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="lblUse">
          <property name="text">
           <string>Utilisé X fois</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QPushButton" name="buttonUpdateName">
            <property name="text">
             <string>Changer le nom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRegenerate">
            <property name="styleSheet">
             <string notr="true">QPushButton {
font-weight: bold;
color:orange;
}</string>
            </property>
            <property name="text">
             <string>Regénérer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonDelete">
            <property name="styleSheet">
             <string notr="true">QPushButton {
font-weight: 600;
color:red;
}</string>
            </property>
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="lblNumPwd">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;En attente de données...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
